src/main/java/openfasttrack/cli/CommandLineInterpreter.java

```diff
@@ -121,19 +121,25 @@ else if (argument.startsWith(SINGLE_CHAR_ARG_PREFIX))
         }
     }
 
-    private void handleSingleCharacterArgument(final ListIterator<String> iterator,
+    private void handleChainedSingleCharacterArguments(final ListIterator<String> iterator,
             final String argument)
     {
-        final String arguments = argument.replaceFirst(SINGLE_CHAR_ARG_PREFIX, "").toLowerCase();
+        final String characters = argument.replaceFirst(SINGLE_CHAR_ARG_PREFIX, "").toLowerCase();
+        final int lastPosition = characters.length() - 1;
 
-        int position = 0;
-        for (final String character : arguments.split(""))
+        for (int position = 0; position <= lastPosition; ++position)
         {
-            ++position;
+            final String character = characters.substring(position, position + 1);
```

Comment:
I thought of using String.charAt(), but you want a String, so this is also fine :)